File: vertx-web-kotlin/src/main/scala/com/perkins/BaseTest.scala
package com.perkins

import org.junit.Test

class BaseTest {
  def main(args: Array[String]): Unit = {
    println("Hello SCALA")
  }


  @Test
  def createStr(): Unit ={
    import org.apache.commons.lang3.RandomStringUtils
    println(RandomStringUtils.randomAlphanumeric(56))
  }

}

File: vertx-web-kotlin/src/main/kotlin/com/perkins/arrow/BaseTest.kt
package com.perkins.arrow

import org.junit.Test

class BaseTest {

    @Test
    fun sayHello(): Unit = println("Hello World")

    fun sayGoodBye(): Unit = println("Good bye World!")

 /*   @Test
    fun greet(): IO<Unit> = IO.fx {
        val pureHello = effect { sayHello() }
        val pureGoodBye = effect { sayGoodBye() }
    }*/
}